daffodil-test/src/test/resources/org/apache/daffodil/section05/simple_types/SimpleTypesUnparse.tdml

```diff
@@ -118,7 +118,7 @@
 
     <xs:element name="float_01" type="xs:float" />
     <xs:element name="double_01" type="xs:double" />
-    <xs:element name="integer_01" type="xs:integer"
+    <xs:element name="integer_01" type="xs:int"
       dfdl:lengthKind="explicit" dfdl:lengthUnits="bits" dfdl:length="8" />
```

Comment:
This is an example of a test schema which wouldn't need any changes under the interpretation that length must be a multiple of 8 bits.